runelite-client/src/main/java/net/runelite/client/plugins/crowdsourcing/varbits/CrowdsourcingVarbits.java

```diff
@@ -93,6 +93,10 @@ public void startUp()
 			{
 				return false;
 			}
+			if (client.getGameState() == GameState.STARTING || client.getGameState() == GameState.UNKNOWN)
```

Comment:
This feels like a hack, but I'm not sure how to tell if `client.getIndexConfig()` will throw an NPE otherwise.

Reply:
Oh, it's actually happening in `client.getVarbit`, not `client.getIndexConfig`. I can't see this part of the code afaik, so not sure what exactly is happening, but it's not the `client` that's null. Here's the exception if I remove the gamestate check and run it:
```
2021-06-06 12:27:56 [Client] WARN  n.r.client.callback.ClientThread - Exception in invoke
oz: null
	at fh.n(fh.java:103)
	at mv.n(mv.java:31)
	at ge.v(ge.java:24)
	at client.bp(client.java)
	at client.xd(client.java:43892)
	at client.getVarbit(client.java:59445)
	at net.runelite.client.plugins.crowdsourcing.varbits.CrowdsourcingVarbits.lambda$startUp$0(CrowdsourcingVarbits.java:100)
	at net.runelite.client.callback.ClientThread.invoke(ClientThread.java:99)
	at net.runelite.client.callback.Hooks.clientMainLoop(Hooks.java:195)
	at gg.zc(gg.java:9184)
	at client.ad(client.java)
	at af.ix(af.java:395)
	at af.run(af.java:349)
	at java.base/java.lang.Thread.run(Thread.java:829)
Caused by: java.lang.NullPointerException: null
	at mv.n(mv.java:32)
	... 12 common frames omitted
```